script/tool/lib/src/common.dart

```diff
@@ -472,14 +476,40 @@ abstract class PluginCommand extends Command<void> {
         packages.add(pathComponents[packagesIndex + 1]);
       }
     }
-    if (packages.isNotEmpty) {
-      final String changedPackages = packages.join(',');
-      print(changedPackages);
-    } else {
+    if (packages.isEmpty) {
       print('No changed packages.');
+    } else {
+      final String changedPackages = packages.join(',');
+      print('Changed packages: $changedPackages');
     }
     return packages;
   }
+
+  // Returns true if one or more files changed that have the potential to affect
+  // any plugin (e.g., CI script changes).
+  Future<bool> _changesRequireFullTest() async {
```

Comment:
Another thought would be to let Cirrus trigger tasks based on what files changed since it already has that functionality and we don't need to reinvent the wheel.  So have a "Run all tests" and "Run changed packages" respective tasks that run inverse to each other
```yml
skip: !changesInclude('.cirrus.yml', '.ci.yaml', '.clang-format', ...)
```
```yml
skip: changesInclude('.cirrus.yml', '.ci.yaml', '.clang-format', ...)
```